packages/sdk-core/src/modules/rateLimiter/RateLimitWatcher.ts

```diff
@@ -0,0 +1,60 @@
+import { TimeHelper } from '../../common/TimeHelper';
+
+export class RateLimitWatcher {
+    /**
+     * Time the single report can stay in the queue.
+     */
+    public MAXIMUM_TIME_IN_QUEUE = 60;
```

Comment:
This is not a constant, is it? Why it is uppercase? Or maybe this should be `readonly` and `private`?

Reply:
it should be readonly. Thanks. The constant can be visible so we can do nice tests around this